contents/handbook/people/team-structure/platform/objectives.mdx

```diff
@@ -1,11 +1,30 @@
-* **Proposed Objective**:
-    * PostHog Cloud in the US remains reliable and launch EU Cloud
-* **Proposed Key Results**:
-    * Multiple organizations using EU hosted PostHog Cloud without regressions in either EU or US cloud
-    * Reliable monitoring and alerting on all key services
-    * Upgrade to Kubernetes 1.25 to unblock key customers
-    * Meet SOC 2 compliance requirements for all cloud infrastructure
-* **Rationale**:
-    * To nail self-serve we need to make it as easy as possible for customers to sign up and get started
-    * For businesses in the EU today, the main barrier is needing to self host to keep data in the EU, this limits trust in our brand and the potential adoption we can drive
-    * Shipping a secure cloud in the EU will also encourage many businesses to get started with PostHog Cloud, even if they ultimately intend to self-host
\ No newline at end of file
+- Objective: High uptime and increase the resilience/redundancy
+    - Key results:
+      - Migrate postgres out of heroku
+      - Regular synthetic load testing on production
+    - Why?
+        - A lack of reliability will cause churn.
+- Objective: Meet the compliance needs of 5 companies that we cannot meet today
+    - Key result: EU Cloud is successfully managed
+    - Key result: Progress made on SOC2
```

Comment:
@ellie is there a specific SOC2 milestone we'd like to aim for? cc @charlescook-ph

Reply:
Discussed with @timgl, we need to figure out what we actually need to change for SOC2 - it might be a very small amount but it may also be a lot and/or have a long tail, so we don't want to commit to 100% completing it next quarter